test/contrib/rails/default_service_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'helper'
+
+require 'contrib/rails/test_helper'
+
+class TracingControllerTest < ActionController::TestCase
+  setup do
+    @original_tracer = Rails.configuration.datadog_trace[:tracer]
+    @tracer = get_test_tracer
+    Rails.configuration.datadog_trace[:tracer] = @tracer
```

Comment:
I think you may want to update our settings so that Rails is re-initialized. You may need to do something like:
```ruby
update_config(:tracer, @tracer)
```

[update_config][1] ref.

[1]: https://github.com/DataDog/dd-trace-rb/blob/a269737649735b9a024b905c0aace3771d6957a7/test/helper.rb#L157